Documentation/Developer/Events/Index.rst

```diff
@@ -132,3 +140,23 @@ events:
 
    This event is triggered when the payment or shipping of an order is edited
    in the backend.
+
+.. confval:: `Extcode\Cart\Event\Session\AfterRestoreAddressEvent`
+
+   This event is triggered in the `SessionHandler` and can be used to validate
+   or change the address after restoring the data them form the session.
```

Comment:
"...them form the session" should be "... from the session"

Reply:
Should be fixed.